### ISSUE TITLE
Touch Control in Live Preview Window

### ISSUE THREAD

**Issue by ChrisTello** [poipoi300/auto-sd-krita-ext-xl#8]
Is it possible to get touch control in the live preview window? Currently you need to use the arrow keys, side scroll bars, or the mouse scroll wheel. I use this with a touch screen and it be great to be able to interact with the window like the canvas area.

**Comment by poipoi300** [comment]
You're asking if it's possible to add touch controls for scrolling, correct? Are you asking for live preview specifically because it's the only one that doesn't have it, or it's the one you need it with the most? 

I think it should be possible, but it's not a priority for me unfortunately.

**Comment by ChrisTello** [comment]
yeah i tend to look at the live preview to see how the generation is coming along. i have a 4k screen and when working with smaller sizes the preview is very small and there is no way to zoom into either, when i work on upscaling the live preview can be very big and i need to maximize the live preview window to view it or try to scroll to the part i wanna see with the side bars.

**Comment by poipoi300** [comment]
Hm, I think live preview being displayed on the canvas would be the better fix to your issue then. This is also something that would interest me. I'll look into it.

For now, there's [this project](https://github.com/DrCyanide/cyanic-sd-krita) that does this (preview in canvas) according to a comment by the author. I don't think it has nearly as many features or is as powerful right now, but hopefully it'll get there.
